content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,357 @@ schema: olm.channel
+- `olm.composite.catalogs`
+    - This schema defines a catalog and the constraints for building the catalog
+
+The `olm.composite.catalogs` cue schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+The `olm.composite` cue schema is:
+```
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+	Config: _ @go(,json.RawMessage)
+}
+```
+
+### Examples
+**Catalog Config File**
+
+The following example creates a catalog configuration that:
+- Defines the `v4.10` catalog and restricts contributions to the catalog to only being `basic` templates
+- Defines the `v4.11` catalog and restricts contributions to the catalog to only being `semver` templates
+- Defines the `v4.12` catalog and restricts contributions to the catalog to only being raw FBC
+- Defines the `v4.13` catalog and restricts contributions to the catalog to use a custom workflow
+
+```yaml
+schema: olm.composite.catalogs
+catalogs:
+- name: v4.10 {{< code_callout 1 >}}
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.24
+    workingDir: catalogs/v4.10 {{< code_callout 2 >}}
+  builders:
+    - olm.builder.basic {{< code_callout 3 >}}
+- name: v4.11
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.25
+    workingDir: catalogs/v4.11
+  builders:
+    - olm.builder.semver {{< code_callout 4 >}}
+- name: v4.12
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.26
+    workingDir: catalogs/v4.12
+  builders:
+    - olm.builder.raw {{< code_callout 5 >}}
+- name: v4.13
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.26
+    workingDir: catalogs/v4.13
+  builders:
+    - olm.builder.custom {{< code_callout 6 >}}
+```
+
+- {{< code_callout 1 >}} Defines the `v4.10` catalog configuration
+- {{< code_callout 2 >}} Defines the directory path that all output should be placed for this catalog. In this example, when rendering using the composite catalog all output for the `v4.10` catalog will be put under the `catalogs/v4.10/` directory
+- {{< code_callout 3 >}} Defines the allowed builders for the `v4.10` catalog. In this example the only allowed builder is the basic builder, meaning the only contributions allowed to be rendered for this catalog are basic templates
+- {{< code_callout 4 >}} Definition for the semver builder, which is used to render semver templates
+- {{< code_callout 5 >}} Definition for the raw builder, which is used to render full FBC (no templates)
```

Comment:
Is it correct to say 'render full FBC' since rendering is how we produce FBC?  Not life-threatening, but maybe "handle raw FBC"?